app/code/Magebit/PageListWidget/Block/Widget/PageList.php

```diff
@@ -30,7 +30,7 @@ class PageList extends Template implements BlockInterface
      * @param array $data
      */
     public function __construct(
-        Template\Context $context,
+        private readonly Template\Context $context,
```

Comment:
Import Context and then write just private readonly Context $context

Then if you do this, you would need to change $context to $this->context in the parent constructor